static/app/utils/replays/hooks/useReplayList.tsx

```diff
@@ -15,6 +15,7 @@ type Options = {
   eventView: EventView;
   location: Location<ReplayListLocationQuery>;
   organization: Organization;
+  enabled?: boolean;
```

Comment:
Given those other nits about bool, another great approach is to not make optional props whenever possible: Then there won't be undefined in the system!

You will have to go around to all the callsites, which in this case there are very few. Some people might say "oh, I'm just concerned with one callsite or situation, therefore optional prop is easier". But really removing optional stuff keeps the calls understandable over time.

```suggestion
  enabled: boolean;
```

Reply:
🫡